util.sh

```diff
@@ -93,6 +129,59 @@ function compose {
         fi
     done
 
+    # docker-compose does not allow to depend on external docker-compose files
+    # which is useful when we want to link to deps from external deps
+    # so we manually "glue" external deps
+    for i in $(
+        _cat_all_compose_files $compose_file \
+            | grep -E '# depends_on:' \
+            | cut -d# -f2
+    ); do
+        IFS=":" read -r service depends_on depends_on_compose_file bar baz < <(echo $i | cut -d: -f2-)
+
+        # very stupid method to detect if we are attemptint to run service
+        if [[ $@ = *"$service"* ]]; then
+            if ! [ -f $depends_on_compose_file ]; then
+                echo -e ${RED}\'$depends_on_compose_file\' is missing to start $depends_on${END_COLOR}
+                exit 1
+            fi
+
+            if ! compose \
+                -f $depends_on_compose_file \
+                ps \
+                --status running \
+                $depends_on \
+                | grep '(healthy)' \
+                    &> /dev/null; then
+
+                if compose \
+                    --project-directory $(dirname $depends_on_compose_file) \
+                    --file $depends_on_compose_file \
+                    --file <(
+                        cat << EOF
+version: "3"
+services:
+    wait_for_$depends_on:
+        image: busybox
+        command: "true"
+        depends_on:
+            $depends_on:
+                condition: service_healthy
+EOF
+                    ) \
```

Comment:
this just adds dummy compose file so that we can leverage compose functionality to wait until that service is healthy. otherwise we need to check health status here with a loop which gets more complicated as you have to account for all the healthcheck params. this seems easier